plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -3674,6 +3674,7 @@ else if (PolygonTag.matches(attribute.getParam())) {
         // @group world
         // @description
         // Returns the current page on display in the book on this Lectern block.
+        // @Deprecated use 'LocationTag.page'
```

Comment:
- `Deprecated` should be lower-case
- Should update the description to just say deprecated with a link to the modern one
- Needs a deprecation warning as well (add a `FutureWarning` to `BukkitImplDeprecations`)